packages/asktug/package.json

```diff
@@ -6,6 +6,7 @@
     "build:watch": "NODE_ENV=development webpack --watch"
   },
   "dependencies": {
+    "@tidb-community/common": "^0.0.0",
```

Comment:
We do not care about the exact versioning for the monorepo dependencies, so it may better to use `*` like we did for `@tidb-community/datasource` and `@tidb-community/ui` below.

Reply:
That was auto added by lerna and I didn't notice that.